Dev Problems/Inventory Mapper/inventory_mapper.rb

```diff
@@ -0,0 +1,66 @@
+class InventoryMapper
+  def initialize
+    @part_id_minimum_quantity_mapping = { "a" => 1, "b" => 1, "c" => 3, "d" => 1, "e" => 4 }
+    @product_name_part_id_mapping = { "Shelf" => ["a"], "Stool" => ["b", "c"], "Table" => ["d", "e"] }
```

Comment:
This and `@part_id_minimum_quantity_mapping` should be constants.

This isn't really scalable with a hash - what happens if there are 100 more products? Or products that share part numbers? You should look into an OO approach